src/test/features/response/routes/check-and-send.ts

```diff
@@ -261,6 +261,7 @@ describe('Defendant response: check and send page', () => {
             claimStoreServiceMock.resolveRetrieveClaimByExternalId()
             claimStoreServiceMock.resolveSaveResponse()
             draftStoreServiceMock.resolveDelete()
+            draftStoreServiceMock.resolveDelete()
```

Comment:
If this keeps growing we could think about adding a `.times(x)` to the resolve functions, so that we won't have to keep duplicating lines

Reply:
As discussed sometime ago,  its only twice and i have moved 1 delete to only delete when feature toggle is set to mediation